### ISSUE TITLE
Incomplete implementation of startover

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#581]
**Reported by valleegr on 24 Nov 2009 15:41 UTC**
From Olivier Lahaye:
```
oscar-config -t does not remove the following packages:

opkg-pfilter-1.707.0-4oscar
lam-oscar-7.1.4-1
perl-Tk-804.026-1.oscar
sge-modulefile-6.0u9-9oscar
ssh-oscar-1.3-1
modules-oscar-3.2.5-2
torque-oscar-2.1.10-4
lam-switcher-modulefile-7.1.4-2.oscar
opkg-pfilter-server-1.707.0-4oscar
oscar-selector-1.2.5-1
oscar-base-server-6.0.5r9065-1
sge-6.0u9-9oscar
modules-default-manpath-oscar-1.0.1-1
atftp-server-0.7-8oscar
mpich-ch_p4-gcc-oscar-module-1.2.7-8
pfilter-1.707-3oscar
torque-oscar-client-2.1.10-4
lam-oscar-modulefile-7.1.4-1
perl-IO-Tty-1.02-4.oscar
pvm-modules-oscar-3.4.5+6-2
systeminstaller-oscar-2.4.7-1
systeminstaller-oscar-x11-2.4.7-1
mpich-ch_p4-gcc-oscar-1.2.7-8
oscar-selector-x11-1.2.5-1
opkg-mtaconfig-1.1.0-1
opkg-pfilter-1.707.0-4oscar
opkg-disable-services-server-1.1.0-1
opkg-netbootmgr-server-1.7.3-1
opkg-networking-1.0.0-2
opkg-opium-1.6-2
opkg-mtaconfig-server-1.1.0-1
opkg-documentation-2.0.b1-1
opkg-ganglia-3.0.9-1
opkg-openmpi-1.2.4-1
opkg-netbootmgr-1.7.3-1
opkg-pvm-3.4.5+6-3
opkg-jobmonarch-0.1.4-1
opkg-opium-server-1.6-2
opkg-jobmonarch-server-0.1.4-1
opkg-pfilter-server-1.707.0-4oscar
opkg-ganglia-server-3.0.9-1
opkg-diskless-1.0-1
opkg-disable-services-1.1.0-1
opkg-networking-server-1.0.0-2
opkg-loghost-server-1.0.0-1
opkg-mpich-server-1.2.7-9
opkg-lam-server-7.1.4-7
opkg-sge-6.1.3-1
opkg-lam-7.1.4-7
opkg-sge-server-6.1.3-1
opkg-openmpi-server-1.2.4-1
opkg-mpich-1.2.7-9
opkg-loghost-1.0.0-1
opkg-pvm-server-3.4.5+6-3
```

**Comment by dikim33** [comment]
**Comment by valleegr on 28 Sep 2010 18:47 UTC**
Right now, we can only remove the OSCAR packages which i think will be enough anyway.

**Comment by dikim33** [comment]
**Comment by valleegr on 28 Sep 2010 19:11 UTC**
(In [9305]) This check-in fixes #581 - Make sure that included OPKGs are removed when starting over.
